CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NEXT
 
+* Your note here!
```

Comment:
☝️ Do we really want that?

Reply:
Good question. I'm fine either way. On the Capistrano project we do this, so I've kind of adopted it for elsewhere.